src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1385,9 +1399,16 @@ class ContentExplorer extends Component<Props, State> {
                 event.preventDefault();
                 break;
             case 'arrowdown':
-                focus(this.rootElement, '.bce-item-row', false);
-                this.setState({ focusedRow: 0 });
-                event.preventDefault();
+                if (viewMode === VIEW_MODE_GRID) {
+                    if (!this.isFocusedOnItem()) {
+                        focus(this.rootElement, '.btn-plain.be-item-label', false);
+                        event.preventDefault();
+                    }
+                } else {
+                    focus(this.rootElement, '.bce-item-row', false);
+                    this.setState({ focusedRow: 0 });
```

Comment:
so ItemList uses a component called KeyBinder from BUIE, that calls `event.stopPropagation()` once you focus the list. ArrowKeyStepper does not call event.stopPropagation, which is why I had to add a check to whether the grid is focused or not.
https://github.com/box/box-ui-elements/blob/master/src/elements/common/KeyBinder.js#L152

and because of the specific line I linked, it cannot be used for ItemGrid.

Reply:
oh I meant I think there's an existing bug for ItemList when using keyboard navigation.

If you press the down arrow, it'll focus on the first element. if you keep pressing down, it'll move down to other rows. <-- that all works as expected

now if you click at the top of the modal (to unfocus the row) and click down, it'll go to the first row. but now if you click down a second time, it'll jump to the previously left off row instead of the second row.

Reply:
oh wait you're right lol, that's weird, should I look into fixing it here? I assume it's because we are setting state for focused row, but not updating the `selected` item

Reply:
I wouldn't fix it in this PR since it's not related to your changes but I'd create a ticket to investigate it later

Reply:
did some more testing and it seems their arrow navigation is slightly broken - instead of showing the "selected" styles, it's showing the "focused" styles

Reply:
it's too much to fix in this ticket, probably